<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>short_job_first_non_preemptive</class>
 <widget class="QDialog" name="short_job_first_non_preemptive">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Short Job First (SJF) non-preemptive</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="process_layout">
     <item>
      <widget class="QLabel" name="process_label">
       <property name="text">
        <string>Process #1:</string>
       </property>
       <property name="buddy">
        <cstring>burst_time_doubleSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="process_no_label">
       <property name="text">
        <string>1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="burst_time_layout">
     <item>
      <widget class="QLabel" name="burst_time_label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Burst Time:</string>
       </property>
       <property name="buddy">
        <cstring>burst_time_doubleSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="burst_time_doubleSpinBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="arrival_time_layout">
     <item>
      <widget class="QLabel" name="arrival_time_label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Arrival Time:</string>
       </property>
       <property name="buddy">
        <cstring>arrival_time_doubleSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="arrival_time_doubleSpinBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="actions_layout">
     <item>
      <widget class="QPushButton" name="back_pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>10</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="next_pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>10</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>206</width>
       <height>96</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="close_pushButton">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>burst_time_doubleSpinBox</tabstop>
  <tabstop>arrival_time_doubleSpinBox</tabstop>
  <tabstop>next_pushButton</tabstop>
  <tabstop>back_pushButton</tabstop>
  <tabstop>close_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>close_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>short_job_first_non_preemptive</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
